Sources/Widgets/Manipulators/PickerManipulator/index.js

```diff
@@ -0,0 +1,69 @@
+import macro from 'vtk.js/Sources/macros';
+import vtkCellPicker from 'vtk.js/Sources/Rendering/Core/CellPicker';
+import vtkAbstractManipulator from 'vtk.js/Sources/Widgets/Manipulators/AbstractManipulator';
+
+// ----------------------------------------------------------------------------
+// vtkPickerManipulator methods
+// ----------------------------------------------------------------------------
+
+function vtkPickerManipulator(publicAPI, model) {
+  // Set our className
+  model.classHierarchy.push('vtkPickerManipulator');
+
+  publicAPI.handleEvent = (callData) => {
+    const { position, pokedRenderer } = callData;
+
+    model.picker.pick([position.x, position.y, 0.0], pokedRenderer);
+    if (model.picker.getActors().length > 0) {
+      model.position = model.picker.getPickedPositions()[0];
+    }
+    return {
+      worldCoords: model.position || null,
+    };
+  };
+
+  macro.setGet(publicAPI, model, ['picker']);
+
+  publicAPI.addActor = (actor) => {
+    model.picker.addPickList(actor);
+  };
+
+  publicAPI.setTolerance = (tolerance) => {
+    model.picker.setTolerance(tolerance);
+  };
```

Comment:
To avoid adding cruft to the PickerManipulator API, I'm thinking we should just drop these methods. Users should configure their picker via `manipulator.getPicker().addActor/setTolerance/etc.`.

Reply:
Ok, changed in [9f6e9ab](https://github.com/Kitware/vtk-js/pull/2907/commits/9f6e9abde6cd3891e82aa9fe41a7b8d7ec1943ec)